app/dashboard/src/components/dashboard/ProjectIcon.tsx

```diff
@@ -74,13 +74,9 @@ export default function ProjectIcon(props: ProjectIconProps) {
   const { user } = authProvider.useFullUserSession()
   const { getText } = textProvider.useText()
 
-  const {
-    data: projectState,
-    isLoading,
-    isError,
-  } = reactQuery.useQuery({
+  const { data: projectState, isError } = reactQuery.useQuery({
```

Comment:
Why did we remove the loading state?

Reply:
it's now always false because of the initial value in the passive listener